cmake/ESP32CommandStation.vgdbproj

```diff
@@ -71,11 +71,10 @@
       <DefaultHeaderDiscoveryMode>HeaderDirectoryAndSubdirectories</DefaultHeaderDiscoveryMode>
       <ESPIDFExtension>
         <IDFCheckout>
-          <Version>master</Version>
-          <Subdirectory>esp-idf/master</Subdirectory>
+          <Version>release/v3.3</Version>
+          <Subdirectory>esp-idf/v3.3</Subdirectory>
```

Comment:
arduino-esp32 is pinned to v3.2 currently and we should stick to the same here for compatibility purposes.

Reply:
done